routes/settings.js

```diff
@@ -16,6 +17,16 @@ router.post('/', async (req, res) => {
     const configs = await ProjectConfigs.all();
 
     const {level, message} = result;
+
+    if (configs.ghost.url && configs.ghost.key) {
+        const ghost = new Ghost();
+        const response = await ghost.registerWebhook();
+        if (response.level === 'error') {
+            res.render('index', response);
```

Comment:
I think we should load the same page & show the error there and then. Makes no sense to navigate to homepage.

Reply:
Done.